Adds information about using openai component to work with Ollama local models (#4756)
dapr/docs: daprdocs/content/en/reference/components-reference/supported-conversation/ollama.md

| `model` | N | The Ollama LLM to use. Defaults to `llama3.2:latest`.  | `phi4:latest` |
| `cacheTTL` | N | A time-to-live value for a prompt cache to expire. Uses Golang duration format.  | `10m` |

### OpenAI Compatibility

> Ollama is compatible with [OpenAI's API](https://ollama.com/blog/openai-compatibility). So we can use the OpenAI component as a drop-in replacement with the following changes:

Reviewer: ```suggestion
Ollama is compatible with [OpenAI's API](https://ollama.com/blog/openai-compatibility). You can use the OpenAI component with Ollama models with the following changes:
```

```yaml
apiVersion: dapr.io/v1alpha1
kind: Component
metadata:
  name: ollama-openai
spec:
  type: conversation.openai # use the openai component type
  metadata:
  - name: key
    value: 'ollama' # just any non-empty string
  - name: model

>     value: gpt-oss:20b  # an ollama model (https://ollama.com/search) in this case openai open source model

Reviewer: ```suggestion
    value: gpt-oss:20b  # an ollama model (https://ollama.com/search) in this case openai open source model. See https://ollama.com/library/gpt-oss
```
